aldryn_redirects/validators.py

```diff
@@ -1,9 +1,8 @@
-# -*- coding: utf-8 -*-
-from __future__ import unicode_literals, print_function, division
+from __future__ import print_function, division
```

Comment:
Import will fail once we add the CI workflow, imports should be ordered alphabetically